Dockerfile

```diff
@@ -7,15 +7,18 @@ RUN apt-get update && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends python3.8-dev python3-pip python3-setuptools openssl libssl-dev gcc pkg-config libffi-dev libxml2-dev libxmlsec1-dev curl && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-COPY . /srv/cartography
 
 # Installs pip supported by python3.8
 RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3.8 get-pip.py
 
+# Installs python dependencies
+COPY setup.py test-requirements.txt ./
 RUN pip install -e . && \
     pip install -r test-requirements.txt
 
+# Install cartography
+COPY . /srv/cartography
```

Comment:
Thanks for taking care to upgrade. Can you explain what this line is for?

Reply:
Before : 
copying source code was done before installing dependencies, so any change will trigger dependencies install

After :
first only requirements are copied, so the docker layer is cached
then files are copied, so change in files wont trigger a new dependencies install